portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,33 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Nikita!");
-  }
+    ArrayList<Comment> comments; 
+
+    public DataServlet() {
+        this.comments = new ArrayList<Comment>();
+    }
+ 
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Gson gson = new Gson();
+        String json = gson.toJson(this.comments);
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+
+    public void doPost(HttpServletRequest request, HttpServletResponse response){
+        String name = request.getParameter("name");
+        String comment = request.getParameter("comment");
+        this.addComment(name, comment);
+        try {
+            response.sendRedirect("/index.html");
+        } catch (IOException e) {
+            System.out.println("Could not redirect");
```

Comment:
does System.out.println actually write this anywhere useful if its in a production environment?

Reply:
Replaced with logger log4j2